assets/xml/objects/object_obj_tokeidai.xml

```diff
@@ -1,59 +1,136 @@
+        <Texture Name="gClockTowerMinuteRingSidesTex" OutName="clock_tower_minute_ring_sides" Format="rgba16" Width="32" Height="16" Offset="0x4E40" />
+
+        <!-- Clock Tower Clock Face Front Textures -->
+        <Texture Name="gClockTowerClockFaceTopTex" OutName="clock_tower_clock_face_top" Format="ci4" Width="64" Height="64" Offset="0x5240" />
+        <Texture Name="gClockTowerClockFaceBottomTex" OutName="clock_tower_clock_face_bottom" Format="ci4" Width="64" Height="64" Offset="0x5A40" />
+        <Texture Name="gClockTowerClockFacePanelBarTex" OutName="clock_tower_clock_face_panel_bar" Format="rgba16" Width="8" Height="8" Offset="0x6240" />
+
+        <!-- Unused Textures-->
+        <Texture Name="gClockTowerUnusedBrickTex" OutName="clock_tower_unused_bricks" Format="rgba16" Width="32" Height="32" Offset="0x62C0" />
+        <Texture Name="gClockTowerUnusedMaskTex" OutName="clock_tower_unused_mask" Format="i4" Width="64" Height="64" Offset="0x6AC0" />
+
+        <!-- Clock Tower Spotlight Beam Textures -->
+        <Texture Name="gClockTowerSpotlightBeamTex" OutName="clock_tower_spotlight_beam" Format="ia8" Width="32" Height="64" Offset="0x72C0" />
+        <Texture Name="gClockTowerSpotlightMaskTex" OutName="clock_tower_spotlight_mask" Format="i8" Width="32" Height="32" Offset="0x7AC0" />
+
+        <!-- Clock Tower Counterweight Mask Texture -->
+        <Texture Name="gClockTowerCounterweightMaskTex" OutName="clock_tower_counterweight_mask" Format="i4" Width="64" Height="64" Offset="0x7EC0" />
+
+        <!-- Clock Tower Unused Tower Wall Texture -->
+        <Texture Name="gClockTowerUnusedTowerWallMaskTex" OutName="clock_tower_unused_tower_wall_mask" Format="i4" Width="64" Height="64" Offset="0x86C0" />
+
+        <!-- Clock Tower Staircase to Rooftop Texture -->
+        <Texture Name="gClockTowerStaircaseToRooftopMaskTex" OutName="clock_tower_staircase_to_rooftop_mask" Format="i4" Width="64" Height="64" Offset="0x8EC0" />
+
+        <!-- Empty DisplayList, probably intended to be used as the xluDList for Clock Tower Termina Field Walls -->
+        <DList Name="gClockTowerUnusedEmptyDL1" Offset="0x9A00" />
+
+        <!-- Clock Tower Termina Field Walls DisplayList -->
+        <DList Name="gClockTowerTerminaFieldWallsDL" Offset="0x9A08" />
+
+        <!-- Clock Tower Termina Field Walls Textures -->
+        <Texture Name="gClockTowerTerminaFieldWallsBrickTex" OutName="clock_tower_termina_field_walls_brick" Format="rgba16" Width="32" Height="32" Offset="0x9B48" />
+        <Texture Name="gClockTowerTerminaFieldWallsMask" OutName="clock_tower_termina_field_walls_mask" Format="i4" Width="64" Height="64" Offset="0xA348" />
+
+        <!-- Clock Tower Spotlight DisplayList -->
+        <DList Name="gClockTowerSpotlightDL" Offset="0xB0C0" />
+
+        <!-- Clock Tower Counterweight DisplayList -->
+        <DList Name="gClockTowerCounterweightDL" Offset="0xB208" />
```

Comment:
A lot of these comments seems redundant. You're just restating the name of the variable and it just adds fluff to the xml files. It'd be nice if you could group them into meaningful, similar assets and comment on those

Reply:
In the past, I actually reordered stuff in the XML to put things into logical blocks. However, it turns out that we're *not* supposed do that, because the ultimate goal is to eventually have the same file splits as the original game. I was made aware of this in the middle of working on this XML, so it sort of ended up a mess as I tried to un-reorder things. Preserving the order *really* doesn't accommodate good comments for this particular object, so I mostly deleted all the comments; I only kept the ones that provide useful info.